subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/buildcache/TaskOutputCachingNativePerformanceTest.groovy

```diff
@@ -43,6 +42,8 @@ class TaskOutputCachingNativePerformanceTest extends AbstractTaskOutputCachingPe
 
         where:
         testProject        | task                         | maxMemory
+        'bigCppApp'        | 'assemble'                   | '256m'
+        'bigCppMulti'      | 'assemble'                   | '1G'
         'nativeDependents' | 'libA0:buildDependentsLibA0' | '3g'
         'bigNative'        | 'assemble'                   | '1G'
```

Comment:
Do we want to keep the old projects, too?

Reply:
I would keep `bigNative`, since it tests C compilation (the new projects are for C++ only). I guess we can remove `nativeDependents`. WDYT?

Reply:
I wouldn't bother with nativeDependents

Reply:
I removed `nativeDependents`.